CHANGELOG.md

```diff
@@ -3,10 +3,14 @@
 *Important!*
 - The next release will be 1.0.0, which means that master will contain breaking changes.
 
+## 0.30.0
+
 *Enchantments*
 - **[Breaking change]** Added PersistentVolumeClaim to the prune whitelist. ([#573](https://github.com/Shopify/kubernetes-deploy/pull/573))
   * To see what resources may be affected, run `kubectl get pvc -o jsonpath='{ range .items[*] }{.metadata.namespace}{ "\t" }{.metadata.name}{ "\t" }{.metadata.annotations}{ "\n" }{ end }' --all-namespaces | grep "last-applied"`
   * To exclude a resource from kubernetes-deploy (and kubectl apply) management, remove the last-applied annotation `kubectl annotate pvc $PVC_NAME kubectl.kubernetes.io/last-applied-configuration-`.
+- Deploying global resources is now disabled by default [#567](https://github.com/Shopify/kubernetes-deploy/pull/567)
```

Comment:
This only applies to the krane version right @dturn ? Kubernetes-deploy emits a warning. I'd either leave this out of the changelog or change it to read something like "Added a warning when kubernetes-deploy is used to deploy global resources. This will be disabled in the Krane version of the command (a separate purpose-built command will be provided)."

Reply:
`Krane deploy` will fail if you try and deploy a global resource.
`exe/kubernetes-deploy` emits a warning if global resources are detected.

`KubernetesDeploy::DeployTask` requires a new param `allow_globals` to keep the old behavior. So we could update this to:

>Deploying global resources directly from KubernetesDeploy::DeployTask is disabled by default, you can use `allow_global: true` to enable the old behavior. This will be disabled in the Krane version of the task, a separate purpose-built task will be provided.

Reply:
```suggestion
- Deploying global resources directly from `KubernetesDeploy::DeployTask` is disabled by default. You can use `allow_globals: true` to enable the old behavior. This will be disabled in the Krane version of the task, and a separate purpose-built task will be provided. [#567](https://github.com/Shopify/kubernetes-deploy/pull/567)
```